tensorboard/tools/migration/src/polymerelementify.ts

```diff
@@ -322,16 +322,20 @@ function getPropsAstNodes(props: ts.PropertyAssignment) {
           ? valueInitializer.initializer
           : undefined;
       }
+      const allowedPropertiesForComputedDecorator = ['type', 'computed'];
```

Comment:
seems like `Set` is more semantically meaningful here.

Reply:
Done